src/js/helpers/disasterHelper.js

```diff
@@ -5,8 +5,10 @@
 
 import { apiRequest } from 'helpers/apiRequest';
 
-const defcAPI = () => apiRequest({
+export const fetchDEFCodes = () => apiRequest({
     url: 'v2/references/def_codes/'
 });
 
-export default defcAPI;
+export const fetchOverview = () => apiRequest({
+    url: 'v2/disaster/overview/'
```

Comment:
Hope we don't need this trailing `/`. I would remove it.

Reply:
We have seen problems in the past when that is not there. Not sure if the backend fixed that but all our routes have it.

Reply:
@tony-sappe can you confirm we can remove this. Playing around in postman, I dont see an issue w/ this on the API side for most end points; ie, the award, psc, naics, tas, agency end points.